content.js

```diff
@@ -6,23 +6,32 @@ function compareRight(s1, s2) {
 var isWikiwand = compareRight(document.location.hostname, 'wikiwand.com');
 
 function replaceUnbalancedBraces(text) {
+  console.log('Balancing ' +  text);
   var stack = [];
   var ret = text.split('');
 
   for (var i = 0; i < text.length; ++i) {
-    if (text[i] == '(') {
+    if (text[i] == '\\') {
+      // handle special cases such as \left( or \right)
```

Comment:
Unfortunately it doesn't fix all formulas, like the one in https://www.wikiwand.com/it/Pi_greco
`{\displaystyle ={\frac {1}{r}}{\int _{-r}^{r}{\sqrt ((\frac {x^{2)){r^{2}-x^{2}}}+1}}\,dx}}` should become
`{\displaystyle ={\frac {1}{r}}{\int _{-r}^{r}{\sqrt {{\frac {x^{2}}{r^{2}-x^{2}}}+1}}\,dx}}` so a couple of `(` should be `{`.
I can't understand why there are these TeX errors in wikiwand